### PR TITLE
Link WebView bug for Web Bluetooth support

### PR DESCRIPTION
**Pull request by foolip**
Link to https://crbug.com/1100993 for all interfaces and navigator.bluetooth.

### CODE REVIEW

File: api/Bluetooth.json [hunk 1]
```diff
@@ -84,7 +84,8 @@
             "version_added": "6.0"
           },
           "webview_android": {
-            "version_added": false
+            "version_added": false,
+            "notes": "See <a href='https://crbug.com/1100993'>bug 1100993</a>"
```
Comment (ddbeck) [inline api/Bluetooth.json:88]: Here and in the other changed lines:

```suggestion
            "notes": "See <a href='https://crbug.com/1100993'>bug 1100993</a>."
```

**Review by ddbeck — CHANGES_REQUESTED** [review]
I have the tiniest possible nit. Thank you for indulging me in this.

File: api/Bluetooth.json (hunk 1 above)
Comment (foolip) [reply api/Bluetooth.json]: I've made the change everywhere and force pushed.

**Review by ddbeck — APPROVED** [review]
Thank you! 🎉

**Pull request merged**